database/data.js

```diff
@@ -0,0 +1,178 @@
+var faker = require('faker');
+var csv = require('csv-parser');
+var fs = require('fs');
+
+//set seed
+faker.seed(123);
+
+var generateRating = function(max){
+  // return random Number between 0 and max
+  var randomNum = parseInt(Math.random() * max)
+  return randomNum;
+};
+
+// generate data
+var data_ratings_ambience = function(){
+  var ratings_ambience = {
+    restaurant_rating : generateRating(5),
+    food_rating : generateRating(5),
+    service_rating : generateRating(5),
+    ambience_rating : generateRating(5),
+    value_rating : generateRating(5),
+    no_of_reviews : faker.random.boolean(),
```

Comment:
Is number of reviews meant to be a boolean?

Reply:
Fixed error